dev-java/openjdk/openjdk-8.442_p06.ebuild

```diff
@@ -156,6 +156,15 @@ src_prepare() {
 		hotspot/make/linux/makefiles/gcc.make || die
 
 	chmod +x configure || die
+
+	# Force gcc because build failed with modern clang, #918655
+	if ! tc-is-gcc; then
```

Comment:
We need to add a BDEPEND on gcc too then (`sys-devel/gcc:*`). The commit message should also reference how it doesn't fully respect CXXFLAGS in various places and that patching that in now is a bit risky given JDK 8 is so old.